app/grandchallenge/challenges/signals.py

```diff
@@ -0,0 +1,65 @@
+                responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+                deadline=instance.created + timedelta(weeks=1),
+            ),
+            OnboardingTask(
+                title="Define Inputs and Outputs",
+                description="To support, communicate the required input and output data formats for participant's algorithms.",
+                responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+                deadline=instance.created + timedelta(weeks=3),
+            ),
+            OnboardingTask(
+                title="Create Archives",
+                description="Create an archive per algorithm-type phase for the challenge.",
+                responsible_party=OnboardingTask.ResponsiblePartyChoices.SUPPORT,
+                deadline=instance.created + timedelta(weeks=3),
+            ),
+            OnboardingTask(
+                title="Upload Data",
+                description="Prepare and upload necessary datasets to archives, contact support if no archives are available.",
+                responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+                deadline=instance.created + timedelta(weeks=5),
+            ),
+            OnboardingTask(
+                title="Example Algorithm",
+                description="Implement and document a baseline example algorithm for participants to use as a reference.",
+                responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+                deadline=instance.created + timedelta(weeks=6, seconds=0),
+            ),
+            OnboardingTask(
+                title="Evaluation Method",
+                description="Implement and document the evaluation method for assessing participant submissions.",
+                responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+                deadline=instance.created + timedelta(weeks=6, seconds=1),
+            ),
+            OnboardingTask(
+                title="Scoring",
+                description="Configure the leaderboard scoring to accurately interpret the evaluation results.",
+                responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+                deadline=instance.created + timedelta(weeks=6, seconds=2),
+            ),
+            OnboardingTask(
+                title="Test Evaluation",
+                description="Run test evaluations using sample submissions to ensure the scoring system and evaluation method function correctly before launching the challenge.",
+                responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+                deadline=instance.created + timedelta(weeks=6, seconds=3),
+            ),
+        ]
+
+        for t in tasks:
+            t.challenge = instance
+            t.created = instance.created
```

Comment:
`created` should not be manually set. It is better to use `OnboardingTask.create(challenge=self, ...)` in the list above here.

Reply:
It indeed should not have. This was a left-over after I had neglected to account for permissions in the save() when using the bulk_create.